content/rest-api/builds.md

```diff
@@ -22,20 +22,22 @@ Note that the workflow and branch information is passed with the curl request wh
 | `branch`      | `string` | Optional. The branch name. Either `branch` or `tag` is **required**. |
 | `tag`         | `string` | Optional. The tag name. Either `branch` or `tag` is **required**. |
 | `environment` | `object` | Optional. Specify environment variables, variable groups, and software versions to override or define in workflow settings. | 
+| `labels`      | `list`   | Optional. Specify labels to be included for the build in addition to existing labels. |
 
 ### Example
 
 ```bash
 curl -H "Content-Type: application/json" -H "x-auth-token: <API Token>" --data '{"appId": "<app_id>","workflowId": "<workflow_id>","branch": "<git_branch_name>"}' https://api.codemagic.io/builds
 ```
 
-#### Pass environment variables and software versions
+#### Pass environment variables, software versions and labels
```

Comment:
alternative
#### Set custom build parameters

for labels let's use `["foo", "bar"]`